README.md

```diff
@@ -138,7 +138,11 @@ $ git clone git@github.com:eclipse/jkube.git
 # 2. Move to Hello World Quickstart folder
 $ cd jkube/quickstarts/maven/hello-world
 
-# 3. Build Project and run JKube goals
+# 3. Configure your local environment to re-use the Docker daemon inside the Minikube instance.
```

Comment:
Shall we add a `Requirements/Prerequisites` section above these steps? I think this demo requires Java8 + Maven + Minikube